"Error: No feed was returned"

### Summary

When importing gsheet, I get "`Error: No feed was returned`"

Mac 10.11.6 (15G31)
Electron app
Both in downloaded .app and in dev environment
### Steps to reproduce

Use `https://docs.google.com/spreadsheets/d/1zeitKHGaHjjTYjr4XvQNxK6Ug7rVSBkXLhdDEgm5yZg/edit#gid=2014102820`
### Expected behavior

Supposed to load a gsheet
### Relevant logs and/or screenshots

This is the console log from handleGsheetsError after printing out `err`
`controller.js:620 Error: No feed was returned
    at /Users/cbodkin/code/dataproofer/electron/node_modules/gsheets/index.js:22:43`
### Possible fixes?

Probably want to catch this error from the gsheets dependency and return a more human readable message. Or patch gsheets.
### Related issues/pull requests?

N/A

--- a/electron/js/controller.js
+++ b/electron/js/controller.js
@@ -617,7 +617,9 @@ function handleSpreadsheet() {
   gsheets.getWorksheetById(gid, "od6", process);
 
   function handleGsheetsError(err) {
-    alert(err.toString());
+    var message = "There was an error downloading the spreadsheet from Google. \
+Please make sure it is shared to public.\n\nDetails:\n" + err.toString();
+    alert(message);
   }
 
   function process(err, sheet) {

